odoo/addons/base/migrations/12.0.1.3/openupgrade_analysis.txt

```diff
@@ -121,6 +121,7 @@ NEW res.country.state: base.state_mn_29
 NEW res.country.state: base.state_mn_32
 NEW res.country.state: base.state_mn_34
 NEW res.country.state: base.state_mn_35
+NEW res.currency: base.MRU
```

Comment:
Why it appears as new? Then why have we put it in noupdate_records.xml

Reply:
What we have put in `noupdate_records.xml` is the change in a `res.country` record.